src/app/main/component/user/components/habit/all-habits/components/habits-list-view/habits-list-view.component.html

```diff
@@ -29,7 +29,7 @@ <h2>{{ habit.habitTranslation.name }}</h2>
         <div class="habit-action-wrp">
           <div>
             <button [disabled]="habit.isAssigned" class="primary-global-button m-btn" (click)="assignStandardHabit()">
-              {{ 'user.habit.all-habits.btn-add' | translate }}
+              {{ habit.isAssigned ? ('user.habit.all-habits.btn-added' | translate) : ('user.habit.all-habits.btn-add' | translate) }}
```

Comment:
it may be better to create a method, but its optional
```getButtonLabel(): string {
  return this.habit.isAssigned 
    ? 'user.habit.all-habits.btn-added' 
    : 'user.habit.all-habits.btn-add';
}
```